ext/include/opentelemetry/ext/http/client/curl/http_client_curl.h

```diff
@@ -366,6 +357,9 @@ class HttpClient : public opentelemetry::ext::http::client::HttpClient
   std::unique_ptr<std::thread> background_thread_;
   std::chrono::milliseconds scheduled_delay_milliseconds_;
 
+  std::chrono::milliseconds background_thread_wait_for_;
+  std::atomic<bool> is_shutdown_;
```

Comment:
```suggestion
  std::atomic<bool> is_shutdown_{false};
```

See #2244 for rationale.

Reply:
With this, then set it in constructure is uncessary.

Reply:
Yes, constructor is fixed as well.